usage/rest-server/src/main/java/brooklyn/rest/resources/ApplicationResource.java

```diff
@@ -82,6 +83,7 @@
 import com.google.common.collect.FluentIterable;
 import com.google.common.collect.Iterables;
 
+@HaHotStateRequired
```

Comment:
Is it right to have these annotations on the Resource rather than the API interface? No strong feelings.